src/ConnectedKubernetes/ChangeLog.md

```diff
@@ -19,6 +19,11 @@
 -->
 ## Upcoming Release
 
+## Version 0.3.0
```

Comment:
This line should be removed because versions will not be updated until we begin the release process.